css/css-fonts/variations/font-style-parsing.html

```diff
@@ -33,7 +33,10 @@
             { style: "oblique 20deg a",   expectedResult: false, message: "'oblique' and angle followed by non-number is invalid" },
             { style: "oblique -",         expectedResult: false, message: "'oblique' followed by isolated minus is invalid" },
             { style: "oblique - 20deg",   expectedResult: false, message: "'oblique' followed by minus and angle separated by space is invalid" },
-            { style: "oblique -a",        expectedResult: false, message: "'oblique' followed by minus and non-number is invalid" }
+            { style: "oblique -a",        expectedResult: false, message: "'oblique' followed by minus and non-number is invalid" },
+            { style: "oblique calc(50deg)",     expectedResult: true,  message: "'oblique' followed by calc is valid" },
+            { style: "oblique calc(-120deg)",   expectedResult: true,  message: "'oblique' followed by calc is valid even if it must be clamped (no computation)" },
```

Comment:
Are you calc(-120deg) is valid value? My understanding is that final calculated value is tested against acceptable range, otherwise it is treated as parse error.

Reply:
https://drafts.csswg.org/css-values/#calc-range

Reply:
Ah, ok. Then both Edge and Chrome have the same bug across font-weight/stretch/style.

Can you review font-weight and font-stretch as well?